main.tf

```diff
@@ -17,13 +17,15 @@ resource "aws_lb_target_group" "this" {
 
   target_type = var.target_group_type
   health_check {
-    enabled  = true
-    protocol = coalesce(var.target_group_health_protocol, var.target_group_protocol)
-    path     = var.target_group_health_path
-    interval = var.target_group_health_interval
-    matcher  = var.target_group_health_response
-    port     = var.target_group_health_port
-    timeout  = var.target_group_health_timeout
+    enabled             = true
+    protocol            = coalesce(var.target_group_health_protocol, var.target_group_protocol)
+    path                = var.target_group_health_path
+    interval            = var.target_group_health_interval
+    matcher             = var.target_group_health_response
+    port                = var.target_group_health_port
+    timeout             = var.target_group_health_timeout
+    healthy_threshold   = var.target_group_hc_healthy_threshold
+    unhealthy_threshold = var.target_group_hc_unhealthy_threshold
```

Comment:
@angellopez-sg have you run `terraform fmt -recursive` for this or just used your IDE formatter? We want to use terraform fmt so it works the same way across IDEs.

Reply:
I always use `terraform fmt -recursive` to have a "prettier" code :)